### PR TITLE
added a webscraping python file

### PR DESCRIPTION
**Pull request by dtkienn**

### CODE REVIEW

File: testt.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+from selenium import webdriver
+import requests
+file = open('pageList.txt')
+videoList = file.readlines()
```
Comment (lthoangg) [inline testt.py:4]: The file should be closed after being used.

File: testt.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+print(videoList)
+
+class CovidImageScraper():
+    def __init__(self):
+        options = webdriver.ChromeOptions()
+        options.add_argument('--ignore-certificate-errors')
+        options.add_argument('--ignore-ssl-errors')
+        self.driver = webdriver.Chrome(chrome_options=options)
+    def getLink(self):
```
Comment (lthoangg) [inline testt.py:13]: You should put _videoList_ in here also.

File: testt.py [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+        self.driver.get(videoList[0])
+    def getPicture(self):
+        all_data = self.driver.find_elements_by_tag_name('img')
+        for data in all_data:
+            print(data.get_attribute("src") + "\n")
```
Comment (lthoangg) [inline testt.py:18]: There are some issues with this function. It doesn't get image links only, it also gets something like this:
`
data:image/jpeg;base64,/9j/4AAQSkZJRgABAQAAAQABAAD/2wCEAAkGBxMTEhUTExMWFRUWFxcVGBcXFhcXFxcWFRgYFxgXFxUYHSggGB0lHRUVITEhJSkrLi4uFx8zODMtNygtLisBCgoKDg0OGhAQGy4lHyQrLTAwLTY3Li0tLSsrNzc3Ny0tKzI2Ky8tLzc3NTI4Li03LTc1MTMwOC4wNS03NTcrK//AABEIAI8BYQMBIgACEQEDEQH/xAAcAAABBQEBAQAAAAAAAAAAAAAFAAMEBgcCAQj/xABQEAABAwICBQYICgULBAMBAAABAAIDBBEFIQYSMUFREyJhcYGRBzI1obGzwdEII0JTc3SSstLwFFJyk+EVJENiY4KDorTC8TM0VKMmhNMl/8QAGAEBAAMBAAAAAAAAAAAAAAAAAAECBAP/xAAtEQEAAgEBBgUCBwEAAAAAAAAAAQIRAwQSMUFRciEyYZGyE3EUM1KSscLiYv/aAAwDAQACEQMRAD8Al+GXT2uoK+KGmmDI3U7JHAxxuu4yStJu5pOxje5QMM8ImIPaCZx+6i/Cg3wjvKcP1SP106EaMPvG1BrbdKauwPK7QPkM4dSHy6XV1zafL6OP8Kh00oLRbhmuZHDWyCANj2nuMsIMVQLbLcjCSermLzCfCJi7ZBy8zXsO4RRA36CGjNWSfBWvALTYkXsePWo8+h8osbMI6HD22QWbD9MZH2+NaeILWg9yexLTB0TdZ0gHCzQb+lVYYE9pzGzpTlcNRnOa49QvvQFW6ZVMjdaMkNGZJY3u2Kv4tprieYinDTx5KI2HTdqmU7i+MaoNtmeW3cvYdH3u4Z8T7rlAEj0wxgkWqx0gwQZj7GSNR6YYhbOf/wBcf4VPk0T5MBxkbnuAN1DraZkRNs9mZQFYdJ6uwJkv/cZ+FAMY04r2F2rMABs+LjPpapNLMCOCrGkDwdY/nJBN8HvhExGpxAwTTh0YimdqiKJubGEtN2tByIUWn8J+IG157X4xQkeZqr3gfN8VPTDU+rK6OGGwsNoQXU6Y4m9gcyqb0fFRWPQebkU7SaX4pa75c+BiiHoagOj+FSNOsbtbvvsPWFYp6pruYBl+e5A+zS2uHjVAv9HGPNq3Uhml9Wf6YfYj/CoDcKba9j2fxTP8nBpyQHn6TVgAPKj7DPwqPTaWVrjnKLfRs/Co0MYLepNNYAUBSu0rqmf0o+wz3KvVfhDrWnKfuijP+1SK6l1rAoBiNFE3bn0f8IC1P4Ra1xA5fP6KP8JVki0irSzXM42bNSOw7Q3NZ4ySEZNIB4DLvJRvD53geMCOHyUD2JeEKvvaOSw2XMcdz2auSjxaeYnvqB+6i/CpU9FHINcNz6CmI6FlgQOsk39iDuTT7EBtnF+HJRX+6puPaSYu6koZaSVofIKjli5sOZZKGx5PblYX2dqq88FnOJV4oqB0lBREOLQBPewHypiRtQVb+XNJgOdK0X2Dkqc3HHxUxUaV6QM8apaOuniH+xXasncA3IG2WY6Bw6jn1cFzHKCLObYd/mKDPB4QMdJsKgH/AOvCP9inYbp1jLvHqR1cjD+BXkYfEdjR6PMmHYLGDe1uxBFw3SvEHePMP3cYv3NReDSOr3v/AMrPYELlDQbD02UmCEfJPtQEZNIqkOtym6/iM9y7OO1Orfldv9Rn4ULli59yL5KRyZ1QEHM+ktXult1sZ+FA8R0uxNvizgf4UR9LURqmtaRc5/ngu4KeOQZ3y6LIKHW6eY8w82pa4dEEOX+ReO8KWMgxttnexPJx3kJFh8izc8+7tvrsLiHyG+lRpmMZmGfZY0edBT4PCxirnMbqg641hYQ31QATbm2Bsb5qVofp/jM9fTRzPPIyTBrhyMQ5lxcawYCNoF0amqIneMy+6xaD2L3Bmt/TKc8iW3mZnqAZ33lBsaSSSD5x+Ed5Th+qR+unVW0Qqebq8CrT8I7ynD9Uj9dOs/wCbVcg0emq3hwDRf3KyRUZFi43Jzy2ZqoYbNcrQKSMOp2/rN9h9xHcgYeXXGqSLKfBymq6R8hFhYZ27hsuk+MRRcq4XPyW329J6FTMWx6Y3BAA83u86A/T4o2R5GtmOO09eSJVBbqDWNr9ZWb4TVEyAlwzPVfPrWkUj7xtudvHoQdRM5g1T7FHNaWu1Q4Z9J/IUvEptSEuG4bugLLqiskD9YWJ27TdBo1dC9lrEhrs7Z7epMTRazbnYoeAaVNna2GcFrhk1x9B6OlF6mAtuLbvyQgDupNVhc1xAzy9yqWPT2jd1K9Y+A2FjAMzcn89et3BZfpPUfFkIHfAmb4oD/Y1Hqyrk1pyAJG/oCpngR8pj6Cf1ZWh4dEBz3bLZD2oGq2URtGsbAWy3nhdA63G3EfF80dQun8eYXu13HmjdfIdar1XWhuTW36TkO7egtWEVzpGgkm44lFY3m9jms7wnGHskGyx3WV3oq/WI3jjvQHKSLLrXnIDWT9G4EEJOYboIdYw3VaxiiLhvVvqrCyC4hTOf7tyCmRYeQcj2EIjEXRjmnPr96brMLe11w09ihMklDt/Ucx50FswGq1iWvFie4n2FSpqfVJG4oZhk7QLOsPQfcj8LS9hJ27D6QgrWJRixKvmj5vh9Lc2ym9a5UTFttld8Bb/APz6T/G9a5BJdq7xf88E2IWk7F2yO5TGJSlgs3bxQeVk4YLj/hBP5WN/G86CYrUvvm+xQd9SC9oDrEi5sfYgvLJGuOds0ZpaVu5Z27F9UC7gSDY8R3K46P4k2VoIO5AaNICV3JALWuumFQ8VmcG5IBlc1gN9YenzKFPXBo25BAMWxB4dtF9yDzVjyTruuDuGxBoGG11yBe44IrNTAjJZ9Q1pZaxGfBWvC8VOVyCOtTiRIZhvODgNnFFcKh+Ojvue30piU7wcjwUzDJfjY+lzfSoF1SSSQfOPwjvKcP1SP106zbDDzlpPwjvKcP1SP106zSgPOQXzR51yez2rTdHoyWuJ8VozWaaJx37T6B/FaQ6d0UGrsa7M8ckFc0jxk6xa0k+jsCqlUJCSTnfjt7hsVixPE4jssScrkfm6rOJwPJB2bd+XRkEHMFK8Zttlw6771pWj8hdGwOA6QRvWcQQyNsW5m43kjNabhcPMjBA2XQStJweQIaM7Hu2LHmOffMuFic87ZZHoW4VZu0ZbQQsVkdJHPJE6wLXusbbQXc3eNxQFMHnYbB/G2tsF1puGVLZIWx7S3Yf6vAlZJRWebW1XXPUbfk/xVz0Zq3RODXAkZDqvt6wgl4zLdxust0gPjDhcdy1fS6lMb+gi4PQVlek4s93Tn+e26Cf4EPKjfoZ/VlaLV82AOPBvvsO1Z34EPKjfoZ/VlaRUgPbqHcPPt83sQVSSodJ42Q3D87Sh1ZSxtN3EbOs36tyk4rKWu1Rlxdx/Z4BCWQk5dO/83QNvkbuDrdnvVl0bqQ7InPZYjeFGocKa5tjkidJhgjuUFrw8WKlBDsLecrouRZAxJHsuoc9twRCcXCiOisgGzg9CGVDW3zaOzJH5mt3oZV07XbPcgHOYx+QFvMf4qwYC0tZqnO97e70oPDh5uBci2efsKOU77WafGQAMZis49OYVw0fP8wpP8f1zlXNJ47WPX6VYcA/7Ck/x/XOQFImb0LxMF19yYiqqyWWeOF1OxkLmM+Mjke5xfG2Qm7XgDxrbNyH14r8wZKbsil//AEWn8NjG9eseETz5xnorkExOhJOZVeko28prXHD8lHammrD4z4dv6j/xoFE8NdI2TxmuHiRyEWc0OGwG2079ytGx3vE/TmLY5RmZx7G91RaujAN9Y2ve2W0q5aEU5AJvkLoMymbOCGEnVDbtcxzTnn8oA5q3aK4aY2Z5EnzLNelqTu2jE9FlpiGShYpGS1EWtUWubkqjJtIYXMkIu2xva47bKtSROe/nEE9XScuhWzT+hkfquF+a6+RtsVEpoJIwdfLftuSCtWy2tWbWrOJ3ZVsstDQMLTdjCQdtlZMIwmB1taCPtYM1W8KqXgc1p1ctoVzwaqJtrWv3dyr+L1/1295Tux0FtFoGtila1oa1tRMA0bANbIAbkeoGWmi/bb6UJ0XHxc31mf7yMUjvjoh/Xb6U2uc61pnqV4Lqkkks6Xzj8I7ynD9Uj9dOsyotq034R3lOH6pH66dZlQi7gOKDTtCIcm/nbmrXpgeVIDHWY0ao4G21DNAaTWIvs2nqGaGYxXySyODObHc9vWgF1dYI8gLnifF/iocOJF51XAlpsLjIjPK3BSJahmbbXI28L9fYhTeUc8Nybc5Zfxugs9KXRu1QCRrDMnItFiPPnfqWhYRHrMBNwbelUbDXu5RsTgCdp3jVA237QtDw+PYBlkgdmOqAOhZ5pZGGvEmpe41T27PP6VouIAAdirmPUolgcCCMtoGzpvuzQZ/AWax1SNa99X2ZZI3h+Ohpax43+Nf02VMiwkgl7Hk2JyuL6w3dSM0cBmsDZrx0+xBoGNOfJAHOzsLA9A/5WVaSC9ncLtP571quFEfoLoXG72OBbfbYixHUPas00ki1dcdqB7wIeVW/Qz/cKu+KVJa8tG05noaqR4D/ACq36Gf7hVv0hkDbv7Ovh2BBCrnR2udp7z1IDVVFjzbD096TdZztYnbvPoAXbqdozPn9yCThb5CRckjZtRiSaRo2mw7kNw6oaw2B7h7FLZWMccyQL26/OgtOCOuASLGyMlt96D4WN4NxYIywoPXiwUR7LlTXDJMSnJBEmp7qC6mKJOJTD3FBGczVFx3HYlBLruAORuMuClNeDty9C5bR2cHt28OKCNpKy8d+Gffe/oCL4B/2FJ/j+uch+LDWheeHouPcVLwZ9qGk/wAf1zkHeAH+cV30sP8Ap4lIrs722qDgUnx9d9JF/p4lziNVq9HStG1eeO2nxhFeCM6E77KsYc0tqKoDYHx3/dgo4+vGdjrdvoCr2GSufUVFhtdHt6IwNoV9n/K1u2PnVE8YE6Ek1E5F9kPdquV1wyKzWi2wKoYRVtZUztNiSIQOwO96vNC+6ptPnjtp8YTCXqKLUhT2hMTtCzpUbSiAlptv6LrNsUDi+zQXEiwLmgC4JJABIystlxeMFuexZ9jFc3lXNbC5xZYE3bbPZkTwWnZt7endiJ8PHM4jH3zHpzRZXsNxCQOsWOdawsLe0q54bXONiKWV3UYva9BaOpFzamdcbefGD23cjMeISQsMhpZAxo1nHXiNhxsH3K0xozM4jTp+/wD2rn1WnRmJ7YXl7CwvmlkDXFpIa91xfVJF+1E6D/rx/tt9K5vzQucPd8fF+230rDq6k6l5tPNaIwviSSS5pfOPwjvKcP1SP106zKhPPC034R3lOH6pH66dU/HKmneKYQBgLbNfZrGknUhz5rW6wvr5uu6+vc2sg1XQ8akRdsAYS48G71V9JHOkceR5sfQM3dXR0Ky4UwDDpichZoPUDcrPKrHDrFrL6uzp7OAQcGSOPx7G3t6vQjGjlZE+RxZHY6pbcgWHGx42zQkUQmAOTduZ9qJUOHhhsx9wNtssygs+jnJOkeWkuPihxHC1wD3K/UbCLdSoGhNEGtvcbzbpO1aHT5WyQN4k3LNCZ2C3jWuLWRbFG5IZNFdp370GY6VYeYJeXjyucwNhO644HYhcWKcqNaPmPGZA3Z7uhXrSeiDoyCLgjZw91lmYjJtLE4a7TquZbfne/wCc0GnaF1HLskJ/6kbDrDiLjMdHRuKpenU1ibK3+D4DnT2s4xujc3hcefMBUXwgv59ulBP8BnlVv0M/3CrTiTNc2PycrdO9VXwGeVW/Qz/cKsON1GrI9u9zif7u7vQCaipAybtG0+73qKyMuO291JFLcknIcPemzVAEAc63DZ3oFTQu17AdGZU6Gk5wzBBPT3KN+k6oOwE715DVvB1rjbwHD/lBf8Iu210eiKr+Gy31epHYggefsTOonyMlAdLmgeMYXBiBXbSvdVBGfS3XodqZfkKVr2UapAP52IGKhmtHIOj07PPZcUr7UVEOio9cU7COaQenu2j7qi1Z1Kajb9aHdOg6wN/x1b9JF/p40xjI18rfwXGBu+NrOmSL1Ea9xKa3NbtWjavPHbT4wivAIfR6ot38UPwfVZPUc4AB8YuTvLB3olLymqS5pPVtKrFM34yfWDjz2ZWP6uwq+z/la3bHzqieMDuGtjNbLY3AMZ67A9HGy07DWi1ws20Zpi6plNrWbFutlYrTMNYQLKm1eeO2nxhMJ7Qm5mJ5q5kCzpBa+AOBCz6vwflJ6ixs5vI2PWwrTKpirFPBeqq8t0Hq3LRo+TU7Y+VUTyUmloXMLrEOO8Hj0ojiT3CknB2ckekd6JYphhadZo1XX28UBxLGo3U08epJrBrmXEbi2423dawzXTYNO99opuxnFq590WnwaKX81MUEv84hH9o30oZSY1HIeSDZWus5wEkT2XDSL2LhnbWHeu8Ll/ncA/tWelZr6d9OcXjCYnLVUkklRL5x+Ed5Th+qR+unWXU3jt/aHpWo/CO8pw/VI/XTrL6Yc9v7Q9KDaZqjVoootgl1nOP7Ng0dtz3KiVlM2I5C43Z+k7lY9NKzkqOma3xyzuDsx6FT6Se4PKHd2nrQdRylziL3ABNgNo/5UmlbMLEi1ze3QPyUOkxMeLEy5O8AnLqHSvS6oPOLJTZlr6rhu3DuQaFoWCC67trshwBWlwOzCx7wezS8vqvbIG2AzaQN28773WwxDMIGccvYWQgSkDbvRfHZtWyEPmaW2KCNiDg9hvu29SyKqvTVEpbm3Wvs2gi9j2laviFg27eH5F1kctaDNKDYtc62Y8Uiw39SC46FV5fURPZkxztR7bbb5Z9INj1Kq+EcWqS3gXeY2RzQKIxVbHf0ZcAf2ieaR1H3IL4Thasd1u7icvMgI+Azyqz6Gf7hVhx2H40v4c3q1d/d6FX/AAGeVWfQzfcKM45Nznxne51/2bnV9HmCADV1xPNHi+nr9ydp4SduQ3W2punpTrZ58P4r2pqhc6p2Db1cEDrmN37zvPsUiKSNw1QRfdkd/TZCGbb3v7boro9Rl8oPyWm56TuCC/UcVg23QjUTtiF0rCCBuRhmxA4TdD5XAIgDkhdVtQPwz5JwVA6kPaSmnyG6AxrAqLKDdQYqkgqdywKBrlrvaNxuO22xMaQ5RUPSKk98rT7VGc/Vmb16w625/wAE7pWbCgHRVdwlZZBBwuS0lVbaZI7fuI0RpKC5udqD4XXQx1FTysscZL47B72tJHIx5gEo7DjlJ/5UH76P8S27RoatrRMVmY3acp/TCsTAkyiHBBcEoQauvFr2kh3cYQUZh0go/wDy6f8AfR/iQzCMIpKueslIZMOWY1r2vJaQIIsgWOsbEu86voaU0pq/ViYiax44/wCq9cfyiZ8Yw7w+ntWVVtzafzterXRNyUGgwWGn1uRjDNYguzcSbZC5cSckUpwsuvet75rwxEe0RHr06rQkNXL10FxIVxShVKAYlgFPK8yPY7XIAJbLKy4bsuGOANro7UvzUR+avp6t9Oc0mYn08ETGVXrNGaUDxX/v5z6Xqs4XgETuVJa4gSytDS99ua+wB52fbtWg1dISDmqjg1O5olIztPMLbL2eVuptm0fSvP1LcY5z6qzWM8E2aTWqoD/Y1F++FSMMd/PKf6VnpUGM3qovopz/AJofcu6KoArqYHfPG3vKza3k0+3+1lo5tmSSSWdL5x+Ed5Th+qR+unWaYYy8rB/WC0v4R3lOH6pH66dZ3o8y9TEP6wQXvGmOnqJIwL6tmMHANaGk991KwrQQGzpzrnbq7GDs39ZVowjCNWZ7yMy4nv3fnpVrioxtKCs0WjTG2swAcAAFOdgrNuqO1WFsSToboBeH0DQRlY5IxyWeSbjp7OBUtm3YgD6SMGV1XXscRdgJN9itmNRXf2KFDTjcgzrGp6hrH6sTiRsGW3vWfR0+trNe0tdvBBBvfp29a+iarC2vB4qr4zo014ILc+NkFCwfEdVwgvm7xTssf47iofhWH89PV7ApNXgT4pmnOzTrA8Q3PVPSNyh+E6XXqWPGx8bHDta1BP8AAb5Vb9DP9wojjTb1L37tna3mn0X7UO8B3lRv0M/qyjGI7JDv1jbrOfoIQCqmpAGr3+5D9UbBv2+5PUlC+U2G87SrbhWAsYASLnieKAJheCPeRe4b0+xX7DMMZEwANXVBSZ3O5FA1ByxgUoRpjVUppQJrMkOrBYlEtZQqllygHAOXojKliJLUQQn0RKiT6zDdGm5LiogBCAFU1TXM1xtHmIzI8yl6TvuMPO4sqndhljPtQqsgMdwPFdlbgdx7dnaiWlDrR4db9SoHZykYQTaGBjs3MaTxLQT50UjoIvm4/sN9yHYWckajVovaOYdgoIvmo/sN9yIQwtaLNaGjbZoAF+Ngo8CltSbWnjIRauowubruNVDgXEq6JTUrUER8YJXLok/ZcPKCE5qr+AwAsn+tVPrCrI8KtyYK9jnmOqmja97pC1oiIDnnWdbWYTt6Vo0ZrNLUtOM468vtlEoVTFq1sP0U/wB6FVs1J/ligZfL9IaSOo29qtYwotkEr55JSGuYNcRgAPLSfEaP1QqJBNfSCjbwnZ5z/BNea4pWs5xGM+PWZ546kPpRJJJZ0vnH4R3lOH6pH66dUbQtl6uPou77IutR8O2i1bVV8UlPTSysFMxhcxpIDhLMSL8bOB7VVdDtB8RjqQ6Sjna2zhcsNswQg1nBWXa1x4DvRuJt1Ew7DJWta0xuFhw3ozBRv3tPcgjaq71bhSDRuJ8UpyOkdwKCBDHzrKQIs1JipCDsKeZTG+xBXsT1jLqt2AC/uUqGAWuRmp8+HEu1gF66kdbYUEExJiphDhntRAUrx8knsTTqV5+Qe5BSNI8MBjdbJ1susbFj2njNV1O05EQtFuFl9CYxhcrm82NxvlayyLwiaHV81Q10VHM9oY0Xaw2uBmgi+BOikbiUb3xvax8M+q4tIa6zDfVcRZ3Yp+Ixl0rWjZa7uu9gO4DvRzwa4diDJ4I56GWKOKKUco5pzdyb2tAyyvrZ3JuQ21s7zqbRyfXcTBLtBzjduAB3IG8Mw1rRe2ZRKKHoUyPDJ/mZPsO9ykx4bL80/wCw73IG4I7BPXTn6DL80/7LvcuhQy/NP+yfcgZCesvf0GX5t/2T7k6KOX5t/wBk+5Ay0JmZTP0SX5t/2T7lw+gl+bf9koIIXVgpX6BL82/7J9y5NBL82/7JQRHNXAfZTf0GX5p/2D7kzLhs3zT/ALDvcgD4hRF5NmkgZmwJsOJts2JnSVl2YeP6lSf/AGsPuRs0M2o5pgkOYI5rhYgEA7M/GKh41hFQ6Ki1YZCWNnDgGm7daRtrjdcC6DjDjsRunch1BhNQBnC8f3SibKKYf0T/ALJQSWOspLHqPDRS72O7k+2lk/Ud3IOrp1jk0aeQ/Id3J+Kmfvae5B0E3I9Ofo7/ANU9yblp3/qnuQMOemJCVI/RZP1HdyQpZP1HdyCJdRahl0RfQyfqO7kw+hl+bd3IAtTwWWUA/wDkNJ9NH94rWazDZ782F52/JKz3D9FK4Y3TzmlmETZmOc8sOqADmSeCD6ESSSQJJJJAkkkkCSSSQeOcALnIDMqJFicTtUBwJcWi3yml7C9oeNrLhp2qTNGHNLTscCD1EWKGQYG1rw/Xc5149Yk3DuTa4NAHyRrP1uy2xAWSSSQJJJJAkkkkCSSSQJJJJAkkkkCSSSQDm43AdYa4u0axG/V1izXA3tuPGGVkRQY6Ox2trO2avRq84Wtx1HuZfpva6MoEkkkgSSSSBJJJIEkkkgSSSSBJJJIGqidrBd2wua3jm9wY0d7gm6eujkcWscHWAddpuLEubk4ZEgscCNy9rablGht7Wcx9/o3tfbqOrbtTNBhjYjzXOIEbImg56rWFxGe/ItH93iSgnJJJIEkkkgSSSSBJJJIP/9k=`

Comment (dtkienn) [reply testt.py:18]: this is an image link, you can try copy and search for it

**Pull request merged**